README.md

```diff
@@ -14,7 +14,7 @@ on any Unix.
 
 Define your rules in
 <br>
-a file as a list of
+a text file as a list of
```

Comment:
Five, seven, then five
syllables make a haiku.
This line is now fixed.

Reply:
I [say](https://en.wikipedia.org/wiki/English-language_vowel_changes_before_historic_/l/#Vile%E2%80%93vial_merger) file as
two syllables, but one seems
more standard I guess?